src/pyodide/internal/workers.py

```diff
@@ -117,15 +139,25 @@ def __init__(
         )
         super().__init__(js_resp.url, js_resp)
 
+    @classmethod
+    def from_response(cls, body: Body, response: "Response") -> "Response":
+        b = body.js_object if isinstance(body, FormData) else body
+        result = cls.__new__(cls)
+        js_resp = js.Response.new(b, response.js_object)
+        super(Response, result).__init__(js_resp.url, js_resp)
+        return result
+
     @staticmethod
     def _create_options(
-        status: HTTPStatus | int = HTTPStatus.OK, statusText="", headers: Headers = None
+        status: HTTPStatus | int = HTTPStatus.OK,
+        status_text="",
+        headers: Headers = None,
     ):
         options = {
             "status": status.value if isinstance(status, HTTPStatus) else status,
         }
-        if len(statusText) > 0:
-            options["statusText"] = statusText
+        if len(status_text) > 0:
```

Comment:
This does the same thing afaict:

```suggestion
        if status_text:
```

Reply:
Explicit is always better than implicit

Reply:
Well it's most common in python code to check if a container is non empty with `if container:`. I think you'll find this suggested in most style guides as the preferred way.